README.md

```diff
@@ -3,6 +3,8 @@
 This README would normally document whatever steps are necessary to get the
 application up and running.
 
+
```

Comment:
_note: Since there is no open PR on the frontend repo for me to review, I'll include my comments for [that project](https://github.com/otmanTR/nice-car-front-and) here.  Review based on the `development` branch._

# Frontend review.

- [ ] Kindly, remove the src/App.test.js which is the test that comes with React.

- [ ] Kindly, remove the `console.log` statements found on `DeleteCar.js` and `AddCar.js` in order to improve the repo's cleanliness. 👍 

- [ ]  In App.js, please, kindly refactor the main `App` function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪

- [ ]  Your frontend project does not contain a proper readme file, which is a requirement for all projects.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), use the template provided, and customize it for your project.  

That way, your app will be well documented 💪

- [ ] Important: As stated in the backend review comments, the migrations do not work for new users on the backend project.  This means I couldn't run the server to check the frontend functionality. This will be done in the next review.

Reply:
Thank you!